kornia/augmentation/container/video.py

```diff
@@ -252,36 +330,7 @@ def forward(
         if params is None:
             params = self.forward_parameters(input.shape)
 
-        # Size of T
-        if self.apply_inverse_func in (InputApplyInverse, MaskApplyInverse):
-            frame_num: int = input.size(self._temporal_channel)
-            input, label = self._input_shape_convert_in(input, label, frame_num)
-        else:
-            if label is not None:
-                raise ValueError(f"Invalid label value. Got {label}")
-            batch_size: int = input.size(0)
-            input = input.view(-1, *input.shape[2:])
-
-        out = super().forward(input, label, params, extra_args=extra_args)
-        if self.return_label:
-            output, label = cast(Tuple[Tensor, Tensor], out)
-        else:
-            output = cast(Tensor, out)
-
-        if isinstance(output, (tuple, list)):
-            if self.apply_inverse_func in (InputApplyInverse, MaskApplyInverse):
-                _out, label = self._input_shape_convert_back(output[0], label, frame_num)
-                output = (_out, output[1])
-            else:
-                if label is not None:
-                    raise ValueError(f"Invalid label value. Got {label}")
-                output = output[0].view(batch_size, -1, *output[0].shape[1:])
-        else:
-            if self.apply_inverse_func in (InputApplyInverse, MaskApplyInverse):
-                output, label = self._input_shape_convert_back(output, label, frame_num)
-            else:
-                if label is not None:
-                    raise ValueError(f"Invalid label value. Got {label}")
-                output = output.view(batch_size, -1, *output.shape[1:])
+        output = self.transform_inputs(input, params, extra_args=extra_args)
+        self._params = params
```

Comment:
Not sure if these two things can be done here, but i think the method does not update the params
```suggestion
        output = self.transform_inputs(input, self._params, extra_args=extra_args)
```